README.md

```diff
@@ -1,7 +1,6 @@
 fastlane-mobile
```

Comment:
What do you think about splitting this file in two for both platforms, having one file in each platform folder and then having this main readme file that redirects to those from each platform?

Reply:
i like the approach